cmd/go-tpc/misc.go

```diff
@@ -61,8 +60,7 @@ func execute(ctx context.Context, w workload.Workloader, action string, index in
 		}
 
 		if err != nil {
-			// For TiDB, we may meet too many conflict errors, so here just ignore it
-			if !silence && !strings.Contains(err.Error(), "conflict") {
+			if !silence {
```

Comment:
May I ask why remove `!strings.Contains(err.Error(), "conflict")`. If optimistic lock, I think we maybe meet this error

Reply:
We could filter the output message outside if we needed, but if we block error info by hard code it may let us miss something important

Reply:
Got